app/src/main/java/org/p2p/wallet/rpc/repository/amount/RpcAmountInMemoryRepository.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.rpc.repository.amount
+
+import java.math.BigInteger
+
+class RpcAmountInMemoryRepository : RpcAmountLocalRepository {
```

Comment:
We don't need that one, I think it's redundant. 
Let's make caching in the repository which makes the same remote request

Reply:
Following our code style we caching any values in InMemoryRepository, even if for now it's redundant for future it will be useful